mne/cov.py

```diff
@@ -582,6 +583,7 @@ def compute_raw_covariance(
     return_estimators=False,
     reject_by_annotation=True,
     rank=None,
+    on_few_samples="warn",
```

Comment:
Nowadays we try to add `*` to functions (and we've been adding them to old functions as we come across them), like
```
    *,
    on_few_samples="warn",
```
But really we could probably go with something much farther up... maybe after `picks` and before `method`?